configure.ac

```diff
@@ -1260,6 +1260,52 @@ AC_ARG_WITH([cryptoauthlib],
     ]
 )
 
+# NXP SE050
+#current configure options line: "./configure --with-se050=/home/pi/Downloads/new_simw_top"
+ 
+ENABLED_SE050="no"
+trylibse050dir=""
+AC_ARG_WITH([se050],
+    [AS_HELP_STRING([--with-se050=PATH],[PATH to SE050 install (default /usr/local/lib/)])],
+    [
+        AC_MSG_CHECKING([for SE050])
+        CPPFLAGS="$CPPFLAGS -DWOLFSSL_SE050"
+        LIBS="$LIBS -lSSS_APIs"
+
+        AC_LINK_IFELSE([AC_LANG_PROGRAM([[#include <fsl_sss_api.h>]], [[ sss_mac_init(0);]])],[ libse050_linked=yes ],[ libse050_linked=no ])
+
+        if test "x$libse050_linked" = "xno" ; then
+            if test "x$withval" != "xno" ; then
+                trylibse050dir=$withval
+            fi
+            if test "x$withval" = "xyes" ; then
+                trylibse050dir="/usr/local/lib/"
+            fi
+
+            LDFLAGS="$LDFLAGS -L$trylibse050dir/simw-top_build/raspbian_native_se050_t1oi2c/sss/"
```

Comment:
Are these paths good for general purpose use? `simw-top_build/raspbian_native_se050_t1oi2c/sss/`

Reply:
Everytime the code is built, it generates that directory. I'm not sure how the end customer will be setting it up but I believe it will work f they build it similarly to the documentation NXP provides. Should /usr/lib or something similar be used instead?

Reply:
If the simw-top supports "make install" then you could use `usr/lib`. Just make sure this is all documented in a README.md in the port/nxp directory.